src/modules/premium-schedules/dto/create-premium-schedule.dto.ts

```diff
@@ -0,0 +1,72 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsDateString, IsInt, IsNotEmpty, IsNumber, IsOptional, IsString, Length, MaxLength } from 'class-validator';
+
+export class CreatePremiumScheduleDto {
+  @IsInt()
+  @IsNotEmpty()
+  @ApiProperty({ example: 10588388, description: 'Can be BS or EW' })
```

Comment:
What are BS and EW?

Reply:
different types of deal (Bond Support and Export Working)

Reply:
thanks @IgnatG!

Reply:
Description was wrong. Updated.

In this project and TFS-API BS (aka BSS) and EW (aka EWCS) are type of Facilities.